internal/handler/queue.go

```diff
@@ -9,10 +10,21 @@ import (
 
 	"github.com/SimplQ/simplQ-golang/internal/datastore"
 	"github.com/SimplQ/simplQ-golang/internal/models"
+	"github.com/go-chi/chi/v5"
 )
 
 func GetQueue(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "GET Queue not implemented")
+	id := r.Context().Value("id").(string)
+	if id == "" {
+		http.Error(w, fmt.Sprintf("Invalid Id: %s", id), 404)
```

Comment:
Could you replace the status code numerical values with constants from the http package: 
`400 -> hhtp.StatusBadRequest, 404 -> http.StatusNotFound` to make it a bit more explicit and less error prone?

Reply:
Done.

Reply:
Hey, did you forget to push your commits? I don't see any changes.

Reply:
Apologies. I have pushed now.